<!doctype html>

<html class="no-js" lang="en">

<head>


	<!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

	Fairlie Reese

	Journal Theme by https://jekyllthemes.io
	Premium + free Jekyll themes for your blog or website.

	- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->


	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">

	<!-- Page Info -->
	<link rel="shortcut icon" href="/images/img/favicon.png">
	<title>Publications – Fairlie Reese</title>
	<meta name="description" content="*These authors contributed equally to this work

">

	<!-- Twitter Card -->
	<meta name="twitter:card" content="summary_large_image">
	<meta name="twitter:title" content="Publications – Fairlie Reese">
	<meta name="twitter:description" content="*These authors contributed equally to this work

">
	<meta name="twitter:image:src" content="http://localhost:4000/images/img/c2c12_rnascope.png">

	<!-- Facebook OpenGraph -->
	<meta property="og:title" content="Publications – Fairlie Reese" />
	<meta property="og:description" content="*These authors contributed equally to this work

" />
	<meta property="og:image" content="http://localhost:4000/images/img/c2c12_rnascope.png" />

	
	<!-- Font Embed Code -->
	<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400|Muli:400,400i,600" rel="stylesheet">
	

	<!-- Styles -->
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<link rel="stylesheet" href="/css/style.css">

	<!-- Icons -->
	<script defer src="https://use.fontawesome.com/releases/v5.1.1/js/solid.js" integrity="sha384-GXi56ipjsBwAe6v5X4xSrVNXGOmpdJYZEEh/0/GqJ3JTHsfDsF8v0YQvZCJYAiGu" crossorigin="anonymous"></script>
	<script defer src="https://use.fontawesome.com/releases/v5.1.1/js/brands.js" integrity="sha384-0inRy4HkP0hJ038ZyfQ4vLl+F4POKbqnaUB6ewmU4dWP0ki8Q27A0VFiVRIpscvL" crossorigin="anonymous"></script>
	<script defer src="https://use.fontawesome.com/releases/v5.1.1/js/fontawesome.js" integrity="sha384-NY6PHjYLP2f+gL3uaVfqUZImmw71ArL9+Roi9o+I4+RBqArA2CfW1sJ1wkABFfPe" crossorigin="anonymous"></script>

	
	<!-- Custom Styles -->
	<style></style>
	

	
	<!-- Analytics Code -->
	
	

	
	<!-- Extra Header JS Code -->
	
	

</head>


<body class="loading ajax-loading" data-site-url="http://localhost:4000" data-page-url="/publications">


	<header class="header">

	<div class="header-image header-image--on" style="background-image: url(/images/img/c2c12_rnascope.png);"></div>
	<div class="header-image"></div>

	<div class="header-overlay"></div>

	<div class="header__content">

		
		<a href="/" class="header__title">
			Fairlie Reese
		</a>
		

		<p class="header__tagline">Bioinformatic analysis · Bioinformatics software/pipeline development · Long-read RNA-seq</p>

		<div class="menu">
			<div class="menu__toggle js-menu-toggle">
				<div class="menu__toggle__icon"><span></span></div>
			</div>
			<div class="menu__wrap">
				<ul class="menu__list">
					
					<li class="menu__list__item">
						<a href="/" class="menu__list__item__link">Blog / Tutorials</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/projects/" class="menu__list__item__link">Projects</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/publications.html" class="menu__list__item__link">Publications</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/about" class="menu__list__item__link">About</a>
					</li>
					
				</ul>
				<ul class="socials">
	
	
	<li class="socials__item">
		
			<a href="https://github.com/fairliereese" target="_blank" class="socials__item__link" title="Github">
				<i class="fab fa-github" aria-hidden="true"></i>
			</a>
		
	</li>
	<!-- console.log(https://github.com/fairliereese) -->
	
	
	
	<li class="socials__item">
		
			<a href="https://orcid.org/0000-0002-9240-0102" target="_blank" class="socials__item__link" title="Orcid">
				<i class="fab fa-orcid" aria-hidden="true"></i>
			</a>
		
	</li>
	<!-- console.log(https://orcid.org/0000-0002-9240-0102) -->
	
	
	
	<li class="socials__item">
		
				<a href="https://scholar.google.com/citations?user=SgA5IcgAAAAJ&hl=en" target="_blank" class="socials__item__link" title="Google Scholar">
					<i class="fas fa-graduation-cap" aria-hidden="true"></i>
				</a>
		
	</li>
	<!-- console.log(https://scholar.google.com/citations?user=SgA5IcgAAAAJ&hl=en) -->
	
	
	
	<li class="socials__item">
		
			<a href="https://twitter.com/FairlieReese" target="_blank" class="socials__item__link" title="Twitter">
				<i class="fab fa-twitter" aria-hidden="true"></i>
			</a>
		
	</li>
	<!-- console.log(https://twitter.com/FairlieReese) -->
	
	
	
	<li class="socials__item">
		
			<a href="https://www.linkedin.com/in/fairlie-reese-a930a5b7/" target="_blank" class="socials__item__link" title="Linkedin">
				<i class="fab fa-linkedin" aria-hidden="true"></i>
			</a>
		
	</li>
	<!-- console.log(https://www.linkedin.com/in/fairlie-reese-a930a5b7/) -->
	
	
</ul>

			</div>
		</div>

	</div>

</header>



	<div class="loader"><svg width="120" height="30" viewBox="0 0 120 30" xmlns="http://www.w3.org/2000/svg"><circle cx="15" cy="15" r="15"><animate attributeName="r" from="15" to="15" begin="0s" dur="0.8s" values="15;9;15" calcMode="linear" repeatCount="indefinite" /><animate attributeName="fill-opacity" from="1" to="1" begin="0s" dur="0.8s" values="1;.5;1" calcMode="linear" repeatCount="indefinite" /></circle><circle cx="60" cy="15" r="9" fill-opacity="0.3"><animate attributeName="r" from="9" to="9" begin="0s" dur="0.8s" values="9;15;9" calcMode="linear" repeatCount="indefinite" /><animate attributeName="fill-opacity" from="0.5" to="0.5" begin="0s" dur="0.8s" values=".5;1;.5" calcMode="linear" repeatCount="indefinite" /></circle><circle cx="105" cy="15" r="15"><animate attributeName="r" from="15" to="15" begin="0s" dur="0.8s" values="15;9;15" calcMode="linear" repeatCount="indefinite" /><animate attributeName="fill-opacity" from="1" to="1" begin="0s" dur="0.8s" values="1;.5;1" calcMode="linear" repeatCount="indefinite" /></circle></svg></div>

	<div class="page-loader"></div>


	<div class="page">

		<div class="page__content" data-page-title="Publications – Fairlie Reese" data-image="/images/img/c2c12_rnascope.png">

			<section class="intro">

	<div class="wrap">

		<h1>Publications</h1>
		<p></p>

	</div>

</section>

<section class="single">

	<div class="wrap">

		<p>*These authors contributed equally to this work</p>

<div class="publications">


  <h2 class="year">2023</h2>
  <ol class="bibliography"><li><div class="col-sm-2 abbr">

  
  <abbr class="badge">In preparation</abbr>
  

</div>
<div id="encode4_rna" class="col-sm-8">
  
    <div class="title">The ENCODE4 long-read RNA-seq collection reveals distinct classes of transcript structure diversity</div>
    <div class="author">
      
        
          
            
              <em>Reese, Fairlie</em>,
            
          
        
      
        
          
            
              
                Williams, Brian,
              
            
          
        
      
        
          
            
              
                Balderrama-Gutierrez, Gabriela,
              
            
          
        
      
        
          
            
              
                Wyman, Dana,
              
            
          
        
      
        
          
            
              
                Çelik, Muhammed Hasan,
              
            
          
        
      
        
          
            
              
                Rebboah, Elisabeth,
              
            
          
        
      
        
          
            
              
                Rezaie, Narges,
              
            
          
        
      
        
          
            
              
                Trout, Diane,
              
            
          
        
      
        
          
            
              
                Razavi-Mohseni, Milad,
              
            
          
        
      
        
          
            
              
                Jiang, Yunzhe,
              
            
          
        
      
        
          
            
              
                Borsari, Beatrice,
              
            
          
        
      
        
          
            
              
                Morabito, Samuel,
              
            
          
        
      
        
          
            
              
                Liang, Heidi Yahan,
              
            
          
        
      
        
          
            
              
                McGill, Cassandra J.,
              
            
          
        
      
        
          
            
              
                Rahmanian, Sorena,
              
            
          
        
      
        
          
            
              
                Sakr, Jasmine,
              
            
          
        
      
        
          
            
              
                Jiang, Shan,
              
            
          
        
      
        
          
            
              
                Zeng, Weihua,
              
            
          
        
      
        
          
            
              
                Carvalho, Klebea,
              
            
          
        
      
        
          
            
              
                Weimer, Annika K.,
              
            
          
        
      
        
          
            
              
                Dionne, Louise A.,
              
            
          
        
      
        
          
            
              
                McShane, Ariel,
              
            
          
        
      
        
          
            
              
                Bedi, Karan,
              
            
          
        
      
        
          
            
              
                Elhajjajy, Shaimae I.,
              
            
          
        
      
        
          
            
              
                Upchurch, Sean,
              
            
          
        
      
        
          
            
              
                Jou, Jennifer,
              
            
          
        
      
        
          
            
              
                Youngworth, Ingrid,
              
            
          
        
      
        
          
            
              
                Gabdank, Idan,
              
            
          
        
      
        
          
            
              
                Sud, Paul,
              
            
          
        
      
        
          
            
              
                Jolanki, Otto,
              
            
          
        
      
        
          
            
              
                Strattan, J. Seth,
              
            
          
        
      
        
          
            
              
                Kagda, Meenakshi S.,
              
            
          
        
      
        
          
            
              
                Snyder, Michael P.,
              
            
          
        
      
        
          
            
              
                Hitz, Ben C.,
              
            
          
        
      
        
          
            
              
                Moore, Jill E.,
              
            
          
        
      
        
          
            
              
                Weng, Zhiping,
              
            
          
        
      
        
          
            
              
                Bennett, David,
              
            
          
        
      
        
          
            
              
                Reinholdt, Laura,
              
            
          
        
      
        
          
            
              
                Ljungman, Mats,
              
            
          
        
      
        
          
            
              
                Beer, Michael A.,
              
            
          
        
      
        
          
            
              
                Gerstein, Mark B.,
              
            
          
        
      
        
          
            
              
                Pachter, Lior,
              
            
          
        
      
        
          
            
              
                Guigó, Roderic,
              
            
          
        
      
        
          
            
              
                Wold, Barbara J.,
              
            
          
        
      
        
          
            
              
                and Mortazavi, Ali
              
            
          
        
      
    </div>
    <div class="periodical">
    
      <em></em>
    
    
      2023
    
    </div>
  
  <div class="links">
  
  
  
  
  
  
  
  
  
  
  </div>
  <!-- Hidden abstract block -->
  
</div>
</li>
<li><div class="col-sm-2 abbr">

  
  <abbr class="badge">bioRXiv</abbr>
  

</div>
<div id="https://doi.org/10.1101/2023.04.04.535623" class="col-sm-8">
  
    <div class="title">The ENCODE Uniform Analysis Pipelines</div>
    <div class="author">
      
        
          
            
              
                Hitz, Benjamin C,
              
            
          
        
      
        
          
            
              
                Lee, Jin-Wook,
              
            
          
        
      
        
          
            
              
                Jolanki, Otto,
              
            
          
        
      
        
          
            
              
                Kagda, Meenakshi S,
              
            
          
        
      
        
          
            
              
                Graham, Keenan,
              
            
          
        
      
        
          
            
              
                Sud, Paul,
              
            
          
        
      
        
          
            
              
                Gabdank, Idan,
              
            
          
        
      
        
          
            
              
                Strattan, J Seth,
              
            
          
        
      
        
          
            
              
                Sloan, Cricket A,
              
            
          
        
      
        
          
            
              
                Dreszer, Timothy,
              
            
          
        
      
        
          
            
              
                Rowe, Laurence D,
              
            
          
        
      
        
          
            
              
                Podduturi, Nikhil R,
              
            
          
        
      
        
          
            
              
                Malladi, Venkat S,
              
            
          
        
      
        
          
            
              
                Chan, Esther T,
              
            
          
        
      
        
          
            
              
                Davidson, Jean M,
              
            
          
        
      
        
          
            
              
                Ho, Marcus,
              
            
          
        
      
        
          
            
              
                Miyasato, Stuart,
              
            
          
        
      
        
          
            
              
                Simison, Matt,
              
            
          
        
      
        
          
            
              
                Tanaka, Forrest,
              
            
          
        
      
        
          
            
              
                Luo, Yunhai,
              
            
          
        
      
        
          
            
              
                Whaling, Ian,
              
            
          
        
      
        
          
            
              
                Hong, Eurie L,
              
            
          
        
      
        
          
            
              
                Lee, Brian T,
              
            
          
        
      
        
          
            
              
                Sandstrom, Richard,
              
            
          
        
      
        
          
            
              
                Rynes, Eric,
              
            
          
        
      
        
          
            
              
                Nelson, Jemma,
              
            
          
        
      
        
          
            
              
                Nishida, Andrew,
              
            
          
        
      
        
          
            
              
                Ingersoll, Alyssa,
              
            
          
        
      
        
          
            
              
                Buckley, Michael,
              
            
          
        
      
        
          
            
              
                Frerker, Mark,
              
            
          
        
      
        
          
            
              
                Kim, Daniel S,
              
            
          
        
      
        
          
            
              
                Boley, Nathan,
              
            
          
        
      
        
          
            
              
                Trout, Diane,
              
            
          
        
      
        
          
            
              
                Dobin, Alex,
              
            
          
        
      
        
          
            
              
                Rahmanian, Sorena,
              
            
          
        
      
        
          
            
              
                Wyman, Dana,
              
            
          
        
      
        
          
            
              
                Balderrama-Gutierrez, Gabriela,
              
            
          
        
      
        
          
            
              <em>Reese, Fairlie</em>,
            
          
        
      
        
          
            
              
                Durand, Neva C,
              
            
          
        
      
        
          
            
              
                Dudchenko, Olga,
              
            
          
        
      
        
          
            
              
                Weisz, David,
              
            
          
        
      
        
          
            
              
                Rao, Suhas S P,
              
            
          
        
      
        
          
            
              
                Blackburn, Alyssa,
              
            
          
        
      
        
          
            
              
                Gkountaroulis, Dimos,
              
            
          
        
      
        
          
            
              
                Sadr, Mahdi,
              
            
          
        
      
        
          
            
              
                Olshansky, Moshe,
              
            
          
        
      
        
          
            
              
                Eliaz, Yossi,
              
            
          
        
      
        
          
            
              
                Nguyen, Dat,
              
            
          
        
      
        
          
            
              
                Bochkov, Ivan,
              
            
          
        
      
        
          
            
              
                Shamim, Muhammad Saad,
              
            
          
        
      
        
          
            
              
                Mahajan, Ragini,
              
            
          
        
      
        
          
            
              
                Aiden, Erez,
              
            
          
        
      
        
          
            
              
                Gingeras, Tom,
              
            
          
        
      
        
          
            
              
                Heath, Simon,
              
            
          
        
      
        
          
            
              
                Hirst, Martin,
              
            
          
        
      
        
          
            
              
                Kent, W James,
              
            
          
        
      
        
          
            
              
                Kundaje, Anshul,
              
            
          
        
      
        
          
            
              
                Mortazavi, Ali,
              
            
          
        
      
        
          
            
              
                Wold, Barbara,
              
            
          
        
      
        
          
            
              
                and Cherry, J Michael
              
            
          
        
      
    </div>
    <div class="periodical">
    
      <em></em>
    
    
      2023
    
    </div>
  
  <div class="links">
  
    <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
  
  
  
    <a href="https://www.biorxiv.org/content/10.1101/2023.04.04.535623v1" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
  
  
    
    <a href="/images/pdf/encode_pipelines_2023.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
    
  
  
  
  
  
  
  
  </div>
  <!-- Hidden abstract block -->
  
  <div class="abstract hidden">
    <p>Abstract The Encyclopedia of DNA elements (ENCODE) project is a collaborative effort to create a comprehensive catalog of functional elements in the human genome. The current database comprises more than 19000 functional genomics experiments across more than 1000 cell lines and tissues using a wide array of experimental techniques to study the chromatin structure, regulatory and transcriptional landscape of the Homo sapiens and Mus musculus genomes. All experimental data, metadata, and associated computational analyses created by the ENCODE consortium are submitted to the Data Coordination Center (DCC) for validation, tracking, storage, and distribution to community resources and the scientific community. The ENCODE project has engineered and distributed uniform processing pipelines in order to promote data provenance and reproducibility as well as allow interoperability between genomic resources and other consortia. All data files, reference genome versions, software versions, and parameters used by the pipelines are captured and available via the ENCODE Portal. The pipeline code, developed using Docker and Workflow Description Language (WDL; https://openwdl.org/ ) is publicly available in GitHub, with images available on Dockerhub ( https://hub.docker.com ), enabling access to a diverse range of biomedical researchers. ENCODE pipelines maintained and used by the DCC can be installed to run on personal computers, local HPC clusters, or in cloud computing environments via Cromwell. Access to the pipelines and data via the cloud allows small labs the ability to use the data or software without access to institutional compute clusters. Standardization of the computational methodologies for analysis and quality control leads to comparable results from different ENCODE collections - a prerequisite for successful integrative analyses. Database URL: https://www.encodeproject.org/</p>
  </div>
  
</div>
</li>
<li><div class="col-sm-2 abbr">

  
  <abbr class="badge">Cell Reports Methods</abbr>
  

</div>
<div id="10.1101/2022.09.22.509094" class="col-sm-8">
  
    <div class="title">High dimensional co-expression networks enable discovery of transcriptomic drivers in complex biological systems</div>
    <div class="author">
      
        
          
            
              
                Morabito, Samuel,
              
            
          
        
      
        
          
            
              <em>Reese, Fairlie</em>,
            
          
        
      
        
          
            
              
                Rahimzadeh, Negin,
              
            
          
        
      
        
          
            
              
                Miyoshi, Emily,
              
            
          
        
      
        
          
            
              
                and Swarup, Vivek
              
            
          
        
      
    </div>
    <div class="periodical">
    
      <em></em>
    
    
      2023
    
    </div>
  
  <div class="links">
  
    <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
  
  
  
    <a href="https://www.biorxiv.org/content/10.1101/2022.09.22.509094v1" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
  
  
    
    <a href="/images/pdf/hdwgcna_2023.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
    
  
  
  
  
  
  
  
  </div>
  <!-- Hidden abstract block -->
  
  <div class="abstract hidden">
    <p>Biological systems are immensely complex, organized into a multi-scale hierarchy of functional units based on tightly-regulated interactions between distinct molecules, cells, organs, and organisms. While experimental methods enable transcriptome-wide measurements across millions of cells, the most ubiquitous bioinformatic tools do not support systems-level analysis. Here we present hdWGCNA, a comprehensive framework for analyzing co-expression networks in high dimensional transcriptomics data such as single-cell and spatial RNA-seq. hdWGCNA provides built-in functions for network inference, gene module identification, functional gene enrichment analysis, statistical tests for network reproducibility, and data visualization. In addition to conventional single-cell RNA-seq, hdWGCNA is capable of performing isoform-level network analysis using long-read single-cell data. We showcase hdWGCNA using publicly available single-cell datasets from Autism spectrum disorder and Alzheimer’s disease brain samples, identifying disease-relevant co-expression network modules in specific cell populations. hdWGCNA is directly compatible with Seurat, a widely-used R package for single-cell and spatial transcriptomics analysis, and we demonstrate the scalability of hdWGCNA by analyzing a dataset containing nearly one million cells.</p>
  </div>
  
</div>
</li>
<li><div class="col-sm-2 abbr">

  
  <abbr class="badge">bioRXiv</abbr>
  

</div>
<div id="https://doi.org/10.1101/2023.05.04.538282" class="col-sm-8">
  
    <div class="title">Long-read sequencing and profiling of RNA-binding proteins reveals the pathogenic mechanism of aberrant splicing of an SCN1A poison exon in epilepsy</div>
    <div class="author">
      
        
          
            
              
                Happ, Hannah C.,
              
            
          
        
      
        
          
            
              
                Schneider, Patricia N,
              
            
          
        
      
        
          
            
              
                Hong, Jung Hwa,
              
            
          
        
      
        
          
            
              
                Goes, Eleanor,
              
            
          
        
      
        
          
            
              
                Bandouil, Masha,
              
            
          
        
      
        
          
            
              
                Biar, Carina G,
              
            
          
        
      
        
          
            
              
                Ramamurthy, Aishwarya,
              
            
          
        
      
        
          
            
              <em>Reese, Fairlie</em>,
            
          
        
      
        
          
            
              
                Engel, Krysta,
              
            
          
        
      
        
          
            
              
                Weckhuysen, Sarah,
              
            
          
        
      
        
          
            
              
                Scheffer, Ingrid E.,
              
            
          
        
      
        
          
            
              
                Mefford, Heather C.,
              
            
          
        
      
        
          
            
              
                Calhoun, Jeffrey D.,
              
            
          
        
      
        
          
            
              
                and Carvill, Gemma L.
              
            
          
        
      
    </div>
    <div class="periodical">
    
      <em></em>
    
    
      2023
    
    </div>
  
  <div class="links">
  
    <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
  
  
  
    <a href="https://www.biorxiv.org/content/10.1101/2023.05.04.538282v1" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
  
  
    
    <a href="/images/pdf/scna1_2023.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
    
  
  
  
  
  
  
  
  </div>
  <!-- Hidden abstract block -->
  
  <div class="abstract hidden">
    <p>Pathogenic loss-of-function SCN1A variants cause a spectrum of seizure disorders. We previously identified variants in individuals with SCN1A-related epilepsy that fall in or near a poison exon (PE) in SCN1A intron 20 (20N). We hypothesized these variants lead to increased PE inclusion, which introduces a premature stop codon, and, therefore, reduced abundance of the full-length SCN1A transcript and Nav1.1 protein. We used a splicing reporter assay to interrogate PE inclusion in HEK293T cells. In addition, we used patient-specific induced pluripotent stem cells (iPSCs) differentiated into neurons to quantify 20N inclusion by long and short-read sequencing and Nav1.1 abundance by western blot. We performed RNA-antisense purification with mass spectrometry to identify RNA-binding proteins (RBPs) that could account for the aberrant PE splicing. We demonstrate that variants in/near 20N lead to increased 20N inclusion by long-read sequencing or splicing reporter assay and decreased Nav1.1 abundance. We also identified 28 RBPs that differentially interact with variant constructs compared to wild-type, including SRSF1 and HNRNPL. We propose a model whereby 20N variants disrupt RBP binding to splicing enhancers (SRSF1) and suppressors (HNRNPL), to favor PE inclusion. Overall, we demonstrate that SCN1A 20N variants cause haploinsufficiency and SCN1A-related epilepsies. This work provides insights into the complex control of RBP-mediated PE alternative splicing, with broader implications for PE discovery and identification of pathogenic PE variants in other genetic conditions</p>
  </div>
  
</div>
</li></ol>

  <h2 class="year">2022</h2>
  <ol class="bibliography"><li><div class="col-sm-2 abbr">

  
  <abbr class="badge">In review</abbr>
  

</div>
<div id="10.1101/2022.03.23.485515" class="col-sm-8">
  
    <div class="title">L-GIREMI uncovers RNA editing sites in long-read RNA-seq</div>
    <div class="author">
      
        
          
            
              
                Liu, Zhiheng,
              
            
          
        
      
        
          
            
              
                Quinones-Valdez, Giovanni,
              
            
          
        
      
        
          
            
              
                Fu, Ting,
              
            
          
        
      
        
          
            
              
                Choudhury, Mudra,
              
            
          
        
      
        
          
            
              <em>Reese, Fairlie</em>,
            
          
        
      
        
          
            
              
                Mortazavi, Ali,
              
            
          
        
      
        
          
            
              
                and Xiao, Xinshu
              
            
          
        
      
    </div>
    <div class="periodical">
    
      <em>bioRxiv</em>
    
    
      2022
    
    </div>
  
  <div class="links">
  
    <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
  
  
  
    <a href="https://www.biorxiv.org/content/10.1101/2022.03.23.485515v1.abstract" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
  
  
    
    <a href="/images/pdf/lgiremi.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
    
  
  
  
  
  
  
  
  </div>
  <!-- Hidden abstract block -->
  
  <div class="abstract hidden">
    <p>Using third-generation sequencers, long-read RNA-seq is increasingly applied in transcriptomic studies given its major advantage in characterizing full-length transcripts. A number of methods have been developed to analyze this new type of data for transcript isoforms and their abundance. Another application, which is significantly under-explored, is to identify and analyze single nucleotide variants (SNVs) in the RNA. Identification of SNVs, such as genetic mutations or RNA editing sites, is fundamental to many biomedical questions. In long-read RNA-seq, SNV analysis presents significant challenges, due to the well-known relatively high error rates of the third-generation sequencers. Here, we present the first study to detect and analyze RNA editing sites in long-read RNA-seq. Our new method, L-GIREMI, effectively handles sequencing errors and biases in the reads, and uses a model-based approach to score RNA editing sites. Applied to PacBio long-read RNA-seq data, L-GIREMI affords a high accuracy in RNA editing identification. In addition, the unique advantage of long reads allowed us to uncover novel insights about RNA editing occurrences in single molecules and double-stranded RNA (dsRNA) structures. L-GIREMI provides a valuable means to study RNA nucleotide variants in long-read RNA-seq.</p>
  </div>
  
</div>
</li>
<li><div class="col-sm-2 abbr">

  
  <abbr class="badge">In review</abbr>
  

</div>
<div id="test" class="col-sm-8">
  
    <div class="title">Relapse to cocaine-seeking is regulated by medial habenula Nr4a2</div>
    <div class="author">
      
        
          
            
              
                Childs, JE,
              
            
          
        
      
        
          
            
              
                Morabito, S,
              
            
          
        
      
        
          
            
              
                Das, S,
              
            
          
        
      
        
          
            
              
                Santelli, C,
              
            
          
        
      
        
          
            
              
                Pham, V,
              
            
          
        
      
        
          
            
              
                Kusche, K,
              
            
          
        
      
        
          
            
              
                Alizo Vera, V,
              
            
          
        
      
        
          
            
              
                Campbell, RR,
              
            
          
        
      
        
          
            
              
                Matheos, DP,
              
            
          
        
      
        
          
            
              <em>Reese, Fairlie</em>,
            
          
        
      
        
          
            
              
                Mortazavi, A,
              
            
          
        
      
        
          
            
              
                Swarup, V,
              
            
          
        
      
        
          
            
              
                and Wood, MA
              
            
          
        
      
    </div>
    <div class="periodical">
    
      <em></em>
    
    
      2022
    
    </div>
  
  <div class="links">
  
    <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
  
  
  
    <a href="" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
  
  
  
  
  
  
  
  
  </div>
  <!-- Hidden abstract block -->
  
  <div class="abstract hidden">
    <p></p>
  </div>
  
</div>
</li>
<li><div class="col-sm-2 abbr">

  
  <abbr class="badge">In review</abbr>
  

</div>
<div id="https://doi.org/10.1101/2022.08.22.504852" class="col-sm-8">
  
    <div class="title">A Python package for weighted gene co-expression network analysis</div>
    <div class="author">
      
        
          
            
              
                Rezaie, Narges,
              
            
          
        
      
        
          
            
              <em>Reese, Fairlie</em>,
            
          
        
      
        
          
            
              
                and Mortazavi, Ali
              
            
          
        
      
    </div>
    <div class="periodical">
    
      <em></em>
    
    
      2022
    
    </div>
  
  <div class="links">
  
    <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
  
  
  
    <a href="https://www.biorxiv.org/content/10.1101/2022.08.22.504852v1" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
  
  
    
    <a href="/images/pdf/pywgcna_2022.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
    
  
  
  
  
  
  
  
  </div>
  <!-- Hidden abstract block -->
  
  <div class="abstract hidden">
    <p>Motivation: Weighted gene co-expression network analysis (WGCNA) is frequently used to identify modules of genes that are co-expressed across many RNA-seq samples. However, the
  current R implementation is slow, not designed to compare modules between multiple WGCNA
  networks, and results are hard to interpret and visualize. We introduce the PyWGCNA Python
  package designed to identify and compare co-expression modules from RNA-seq data.
  Results: We apply PyWGCNA to two distinct datasets of brain bulk RNA-seq from MODEL-AD to
  identify modules associated with the genotypes. We compare the resulting modules to each
  other to find modules with significant overlap across the datasets.
  Availability: The PyWGCNA library for Python 3 is available on PyPi at
  https://pypi.org/project/PyWGCNA and on GitHub at
  https://github.com/mortazavilab/PyWGCNA.</p>
  </div>
  
</div>
</li></ol>

  <h2 class="year">2021</h2>
  <ol class="bibliography"><li><div class="col-sm-2 abbr">

  
  <abbr class="badge">Genome Biology</abbr>
  

</div>
<div id="doi.org/10.1186/s13059-021-02505-w" class="col-sm-8">
  
    <div class="title">Mapping and modeling the genomic basis of differential RNA isoform expression at single-cell resolution with LR-Split-seq</div>
    <div class="author">
      
        
          
            
              
                Rebboah, Elisabeth*,
              
            
          
        
      
        
          
            
              <em>Reese, Fairlie*</em>,
            
          
        
      
        
          
            
              
                Williams, Katherine,
              
            
          
        
      
        
          
            
              
                Balderrama-Gutierrez, Gabriela,
              
            
          
        
      
        
          
            
              
                McGill, Cassandra,
              
            
          
        
      
        
          
            
              
                Trout, Diane,
              
            
          
        
      
        
          
            
              
                Rodriguez, Isaryhia,
              
            
          
        
      
        
          
            
              
                Liang, Heidi,
              
            
          
        
      
        
          
            
              
                Wold, Barbara J.,
              
            
          
        
      
        
          
            
              
                and Mortazavi, Ali
              
            
          
        
      
    </div>
    <div class="periodical">
    
      <em></em>
    
    
      2021
    
    </div>
  
  <div class="links">
  
    <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
  
  
  
    <a href="https://doi.org/10.1186/s13059-021-02505-w" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
  
  
    
    <a href="/images/pdf/lr_split_seq_2021.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
    
  
  
    
    <a href="/images/pdf/lr_split_seq_supp_2021.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Supp</a>
    
  
  
  
  
  
  
  </div>
  <!-- Hidden abstract block -->
  
  <div class="abstract hidden">
    <p>Alternative RNA isoforms are defined by promoter choice, alternative splicing, and polyA site selection. Although differential isoform expression is known to play a large regulatory role in eukaryotes, it has proved challenging to study with standard short-read RNA-seq because of the uncertainties it leaves about the full-length structure and precise termini of transcripts. The rise in throughput and quality of long-read sequencing now makes it possible, in principle, to unambiguously identify most transcript isoforms from beginning to end. However, its application to single-cell RNA-seq has been limited by throughput and expense. Here, we develop and characterize long-read Split-seq (LR-Split-seq), which uses a combinatorial barcoding-based method for sequencing single cells and nuclei with long reads. We show that LR-Split-seq can associate isoforms with cell types with relative economy and design flexibility. We characterize LR-Split-seq for whole cells and nuclei by using the well-studied mouse C2C12 system in which mononucleated myoblast cells differentiate and fuse into multinucleated myotubes. We show that the overall results are reproducible when comparing long- and short-read data from the same cell or nucleus. We find substantial evidence of differential isoform expression during differentiation including alternative transcription start site (TSS) usage. We integrate the resulting isoform expression dynamics with snATAC-seq chromatin accessibility to validate TSS-driven isoform choices. LR-Split-seq provides an affordable method for identifying cluster-specific isoforms in single cells that can be further quantified with companion deep short-read scRNA-seq from the same cell populations.</p>
  </div>
  
</div>
</li>
<li><div class="col-sm-2 abbr">

  
  <abbr class="badge">Genome Research</abbr>
  

</div>
<div id="10.1101/gr.275723.121" class="col-sm-8">
  
    <div class="title">Integration of high-resolution promoter profiling assays reveals novel, cell type-specific transcription start sites across 115 human cell and tissue types</div>
    <div class="author">
      
        
          
            
              
                Moore, Jill E,
              
            
          
        
      
        
          
            
              
                Zhang, Xiao-Ou,
              
            
          
        
      
        
          
            
              
                Elhajjajy, Shaimae I,
              
            
          
        
      
        
          
            
              
                Fan, Kaili,
              
            
          
        
      
        
          
            
              
                Pratt, Henry E,
              
            
          
        
      
        
          
            
              <em>Reese, Fairlie</em>,
            
          
        
      
        
          
            
              
                Mortazavi, Ali,
              
            
          
        
      
        
          
            
              
                and Weng, Zhiping
              
            
          
        
      
    </div>
    <div class="periodical">
    
      <em></em>
    
    
      2021
    
    </div>
  
  <div class="links">
  
    <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
  
  
  
    <a href="https://doi.org/10.1101/gr.275723.121" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
  
  
    
    <a href="/images/pdf/tss_2021.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
    
  
  
  
  
  
  
  
  </div>
  <!-- Hidden abstract block -->
  
  <div class="abstract hidden">
    <p>Accurate transcription start site (TSS) annotations are essential for understanding transcriptional regulation and its role in human disease. Gene collections such as GENCODE contain annotations for tens of thousands of TSSs, but not all of these annotations are experimentally validated, nor do they contain information on cell type-specific usage. Therefore, we sought to generate a collection of experimentally validated TSSs by integrating RNA Annotation and Mapping of Promoters for the Analysis of Gene Expression (RAMPAGE) data from 115 cell and tissue types, which resulted in a collection of approximately 50 thousand representative RAMPAGE peaks. These peaks were primarily proximal to GENCODE-annotated TSSs and were concordant with other transcription assays. Because RAMPAGE uses paired-end reads, we were then able to connect peaks to transcripts by analyzing the genomic positions of the 3’ ends of read mates. Using this paired-end information, we classified the vast majority (37 thousand) of our RAMPAGE peaks as verified TSSs, updating TSS annotations for 20% of GENCODE genes. We also found that these updated TSS annotations were supported by epigenomic and other transcriptomic datasets. To demonstrate the utility of this RAMPAGE rPeak collection, we intersected it with the NHGRI/EBI genome-wide association studies (GWAS) catalog and identified new candidate GWAS genes. Overall, our work demonstrates the importance of integrating experimental data to further refine TSS annotations and provides a valuable resource for the biological community.</p>
  </div>
  
</div>
</li>
<li><div class="col-sm-2 abbr">

  
  <abbr class="badge">Frontiers in Marine Science</abbr>
  

</div>
<div id="doi.org/10.3389/fmars.2021.796849" class="col-sm-8">
  
    <div class="title">Long-Term Patterns of Noise From Underwater Explosions and Their Relation to Fisheries in Southern California</div>
    <div class="author">
      
        
          
            
              
                Krumpel, Anna,
              
            
          
        
      
        
          
            
              
                Rice, Ally,
              
            
          
        
      
        
          
            
              
                Frasier, Kaitlin E.,
              
            
          
        
      
        
          
            
              <em>Reese, Fairlie</em>,
            
          
        
      
        
          
            
              
                Trickey, Jennifer S.,
              
            
          
        
      
        
          
            
              
                Simonis, Anne E.,
              
            
          
        
      
        
          
            
              
                Ryan, John P.,
              
            
          
        
      
        
          
            
              
                Wiggins, Sean M.,
              
            
          
        
      
        
          
            
              
                Denzinger, Annette,
              
            
          
        
      
        
          
            
              
                Schnitzler, Hans-Ulrich,
              
            
          
        
      
        
          
            
              
                and Baumann-Pickering, Simone
              
            
          
        
      
    </div>
    <div class="periodical">
    
      <em></em>
    
    
      2021
    
    </div>
  
  <div class="links">
  
    <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
  
  
  
    <a href="https://doi.org/10.3389/fmars.2021.796849" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
  
  
    
    <a href="/images/pdf/seal_bomb_2021.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
    
  
  
  
  
  
  
  
  </div>
  <!-- Hidden abstract block -->
  
  <div class="abstract hidden">
    <p>Acoustic deterrents can reduce marine mammal interactions with fisheries and aquacultures, but they contribute to an increasing level of underwater noise. In Southern California, commercially produced explosive deterrents, commonly known as “seal bombs,” are used to protect fishing gear and catch from pinniped predation, which can cause extensive economic losses for the fishing community. Passive acoustic monitoring data collected between 2005 and 2016 at multiple sites within the Southern California Bight and near Monterey Bay revealed high numbers of these small-charge underwater explosions, long-term, spatio-temporal patterns in their occurrence, and their relation to different commercial purse-seine fishing sectors. The vast majority of explosions occurred at nighttime and at many nearshore sites high explosion counts were detected, up to 2,800/day. Received sound exposure levels of up to 189 dB re 1 μPa2-s indicate the potential for negative effects on marine life, especially in combination with the persistence of recurring explosions during periods of peak occurrence. Due to the highly significant correlation and similar spatio-temporal patterns of market squid landings and explosion occurrence at many sites, we conclude that the majority of the recorded explosions come from seal bombs being used by the California market squid purse-seine fishery. Additionally, seal bomb use declined over the years of the study, potentially due to a combination of reduced availability of market squid driven by warm water events in California and regulation enforcement. This study is the first to provide results on the distribution and origin of underwater explosions off Southern California, but there is a substantial need for further research on seal bomb use in more recent years and their effects on marine life, as well as for establishing environmental regulations on their use as a deterrent.</p>
  </div>
  
</div>
</li>
<li><div class="col-sm-2 abbr">

  
  <abbr class="badge">In review</abbr>
  

</div>
<div id="10.21203/rs.3.rs-777702/v1" class="col-sm-8">
  
    <div class="title">Systematic assessment of long-read RNA-seq methods for transcript identification and quantification</div>
    <div class="author">
      
        
          
            
              
                Pardo-Palacios, Francisco*,
              
            
          
        
      
        
          
            
              <em>Reese, Fairlie*</em>,
            
          
        
      
        
          
            
              
                Carbonell-Sala, Silvia*,
              
            
          
        
      
        
          
            
              
                Diekhans, Mark*,
              
            
          
        
      
        
          
            
              
                Liang, Cindy*,
              
            
          
        
      
        
          
            
              
                Wang, Dingjie*,
              
            
          
        
      
        
          
            
              
                Williams, Brian*,
              
            
          
        
      
        
          
            
              
                Adams, Matthew,
              
            
          
        
      
        
          
            
              
                Behera, Amit,
              
            
          
        
      
        
          
            
              
                Lagarde, Julien,
              
            
          
        
      
        
          
            
              
                Li, Haoran,
              
            
          
        
      
        
          
            
              
                Prjibelski, Andrey,
              
            
          
        
      
        
          
            
              
                Balderrama-Gutierrez, Gabriela,
              
            
          
        
      
        
          
            
              
                Çelik, Muhammed Hasan,
              
            
          
        
      
        
          
            
              
                María, Maite De,
              
            
          
        
      
        
          
            
              
                Denslow, Nancy,
              
            
          
        
      
        
          
            
              
                Garcia-Reyero, Natàlia,
              
            
          
        
      
        
          
            
              
                Goetz, Stefan,
              
            
          
        
      
        
          
            
              
                Hunter, Margaret,
              
            
          
        
      
        
          
            
              
                Loveland, Jane,
              
            
          
        
      
        
          
            
              
                Menor, Carlos,
              
            
          
        
      
        
          
            
              
                Moraga, David,
              
            
          
        
      
        
          
            
              
                Mudge, Jonathan,
              
            
          
        
      
        
          
            
              
                Takahashi, Hazuki,
              
            
          
        
      
        
          
            
              
                Tang, Alison,
              
            
          
        
      
        
          
            
              
                Youngworth, Ingrid,
              
            
          
        
      
        
          
            
              
                Carninci, Piero,
              
            
          
        
      
        
          
            
              
                Guigó, Roderic,
              
            
          
        
      
        
          
            
              
                Tilgner, Hagen,
              
            
          
        
      
        
          
            
              
                Wold, Barbara,
              
            
          
        
      
        
          
            
              
                Vollmers, Christopher,
              
            
          
        
      
        
          
            
              
                Sheynkman, Gloria,
              
            
          
        
      
        
          
            
              
                Frankish, Adam,
              
            
          
        
      
        
          
            
              
                Au, Kin Fai,
              
            
          
        
      
        
          
            
              
                Conesa, Ana,
              
            
          
        
      
        
          
            
              
                Mortazavi, Ali,
              
            
          
        
      
        
          
            
              
                and Brooks, Angela
              
            
          
        
      
    </div>
    <div class="periodical">
    
      <em></em>
    
    
      2021
    
    </div>
  
  <div class="links">
  
    <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
  
  
  
    <a href="https://doi.org/10.21203/rs.3.rs-777702/v1" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
  
  
    
    <a href="/images/pdf/lrgasp_2021.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
    
  
  
  
  
  
  
  
  </div>
  <!-- Hidden abstract block -->
  
  <div class="abstract hidden">
    <p>With increased usage of long-read sequencing technologies to perform transcriptome analyses, there becomes a greater need to evaluate different methodologies including library preparation, sequencing platform, and computational analysis tools. Here, we report the study design of a community effort called the Long-read RNA-Seq Genome Annotation Assessment Project (LRGASP) Consortium, whose goals are characterizing the strengths and remaining challenges in using long-read approaches to identify and quantify the transcriptomes of both model and non-model organisms. The LRGASP organizers have generated cDNA and direct RNA datasets in human, mouse, and manatee samples using different protocols followed by sequencing on Illumina, Pacific Biosciences, and Oxford Nanopore Technologies platforms. Participants will use the provided data to submit predictions for three challenges: transcript isoform detection with a high-quality genome, transcript isoform quantification, and de novo transcript isoform identification. Evaluators from different institutions will determine which pipelines have the highest accuracy for a variety of metrics using benchmarks that include spike-in synthetic transcripts, simulated data, and a set of undisclosed, manually curated transcripts by GENCODE. We also describe plans for experimental validation of predictions that are platform-specific and computational tool-specific. We believe that a community effort to evaluate long-read RNA-seq methods will help move the field toward a better consensus on the best approaches to use for transcriptome analyses.</p>
  </div>
  
</div>
</li></ol>

  <h2 class="year">2020</h2>
  <ol class="bibliography"><li><div class="col-sm-2 abbr">

  
  <abbr class="badge">Bioinformatics</abbr>
  

</div>
<div id="doi.org/10.1093/bioinformatics/btaa836" class="col-sm-8">
  
    <div class="title">Swan: a library for the analysis and visualization of long-read transcriptomes</div>
    <div class="author">
      
        
          
            
              <em>Reese, Fairlie</em>,
            
          
        
      
        
          
            
              
                and Mortazavi, Ali
              
            
          
        
      
    </div>
    <div class="periodical">
    
      <em></em>
    
    
      2020
    
    </div>
  
  <div class="links">
  
    <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
  
  
  
    <a href="https://doi.org/10.1093/bioinformatics/btaa836" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
  
  
    
    <a href="/images/pdf/swan_2020.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
    
  
  
  
  
  
  
  
  </div>
  <!-- Hidden abstract block -->
  
  <div class="abstract hidden">
    <p>Motivation
    Long-read RNA-sequencing technologies such as PacBio and Oxford Nanopore have discovered an explosion of new transcript isoforms that are difficult to visually analyze using currently available tools. We introduce the Swan Python library, which is designed to analyze and visualize transcript models.

    Results
    Swan finds 4909 differentially expressed transcripts between cell lines HepG2 and HFFc6, including 279 that are differentially expressed even though the parent gene is not. Additionally, Swan discovers 285 reproducible exon skipping and 47 intron retention events not recorded in the GENCODE v29 annotation.

    Availability and implementation
    The Swan library for Python 3 is available on PyPi at https://pypi.org/project/swan-vis/ and on GitHub at https://github.com/mortazavilab/swan_vis.</p>
  </div>
  
</div>
</li>
<li><div class="col-sm-2 abbr">

  
  <abbr class="badge">In review</abbr>
  

</div>
<div id="doi.org/10.1101/672931" class="col-sm-8">
  
    <div class="title">A technology-agnostic long-read analysis pipeline for transcriptome discovery and quantification</div>
    <div class="author">
      
        
          
            
              
                Wyman, Dana*,
              
            
          
        
      
        
          
            
              
                Balderrama-Gutierrez, Gabriela*,
              
            
          
        
      
        
          
            
              <em>Reese, Fairlie*</em>,
            
          
        
      
        
          
            
              
                Jiang, Shan,
              
            
          
        
      
        
          
            
              
                Rahmanian, Sorena,
              
            
          
        
      
        
          
            
              
                Forner, Stefania,
              
            
          
        
      
        
          
            
              
                Matheos, Dina,
              
            
          
        
      
        
          
            
              
                Zend, Weihua,
              
            
          
        
      
        
          
            
              
                Williams, Brian,
              
            
          
        
      
        
          
            
              
                Trout, Diane,
              
            
          
        
      
        
          
            
              
                England, Whitney,
              
            
          
        
      
        
          
            
              
                Chu, Shu-Hui,
              
            
          
        
      
        
          
            
              
                Spitale, Robert C.,
              
            
          
        
      
        
          
            
              
                Tenner, Andrea J.,
              
            
          
        
      
        
          
            
              
                Wold, Barbara J.,
              
            
          
        
      
        
          
            
              
                and Mortazavi, Ali
              
            
          
        
      
    </div>
    <div class="periodical">
    
      <em></em>
    
    
      2020
    
    </div>
  
  <div class="links">
  
    <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
  
  
  
    <a href="https://doi.org/10.1101/672931" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
  
  
    
    <a href="/images/pdf/talon_2020.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
    
  
  
    
    <a href="/images/pdf/talon_supp_2020.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Supp</a>
    
  
  
  
  
  
  
  </div>
  <!-- Hidden abstract block -->
  
  <div class="abstract hidden">
    <p>Alternative splicing is widely acknowledged to be a crucial regulator of gene expression and is a key contributor to both normal developmental processes and disease states. While cost-effective and accurate for quantification, short-read RNA-seq lacks the ability to resolve full-length transcript isoforms despite increasingly sophisticated computational methods. Long-read sequencing platforms such as Pacific Biosciences (PacBio) and Oxford Nanopore (ONT) bypass the transcript reconstruction challenges of short reads. Here we introduce TALON, the ENCODE4 pipeline for platform-independent analysis of long-read transcriptomes. We apply TALON to the GM12878 cell line and show that while both PacBio and ONT technologies perform well at full-transcript discovery and quantification, each displayed distinct technical artifacts. We further apply TALON to mouse hippocampus and cortex transcriptomes and find that 422 genes found in these regions have more reads associated with novel isoforms than with annotated ones. We demonstrate that TALON is a capable of tracking both known and novel transcript models as well as their expression levels across datasets for both simple studies and in larger projects. These properties will enable TALON users to move beyond the limitations of short-read data to perform isoform discovery and quantification in a uniform manner on existing and future long-read platforms.</p>
  </div>
  
</div>
</li></ol>

  <h2 class="year">2019</h2>
  <ol class="bibliography"><li><div class="col-sm-2 abbr">

  
  <abbr class="badge">RNA</abbr>
  

</div>
<div id="10.1261/rna.070987.119" class="col-sm-8">
  
    <div class="title">Exon Size and Sequence Conservation Improves Identification of Splice Altering Nucleotides</div>
    <div class="author">
      
        
          
            
              
                Movassat, Maliheh,
              
            
          
        
      
        
          
            
              
                Forouzmand, Elmira,
              
            
          
        
      
        
          
            
              <em>Reese, Fairlie</em>,
            
          
        
      
        
          
            
              
                and Hertel, Klemens J
              
            
          
        
      
    </div>
    <div class="periodical">
    
      <em></em>
    
    
      2019
    
    </div>
  
  <div class="links">
  
    <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
  
  
  
    <a href="https://doi.org/10.1261/rna.070987.119" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
  
  
    
    <a href="/images/pdf/exon_cons_2019.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
    
  
  
  
  
  
  
  
  </div>
  <!-- Hidden abstract block -->
  
  <div class="abstract hidden">
    <p>Pre-mRNA splicing is regulated through multiple trans-acting splicing factors. These regulators interact with the pre-mRNA at intronic and exonic positions. Given that most exons are protein coding, the evolution of exons must be modulated by a combination of selective coding and splicing pressures. It has previously been demonstrated that selective splicing pressures are more easily deconvoluted when phylogenetic comparisons are made for exons of identical size, suggesting that exon size-filtered sequence alignments may improve identification of nucleotides evolved to mediate efficient exon ligation. To test this hypothesis, an exon size database was created, filtering 76 vertebrate sequence alignments based on exon size conservation. In addition to other genomic parameters, such as splice site strength, gene position or flanking intron length, this database permits identification of exons that are size and/or sequence conserved. Highly size-conserved exons are always sequence conserved. However, sequence conservation does not necessitate exon size conservation. Our analysis identified evolutionarily young exons and demonstrated that length conservation is a strong predictor of alternative splicing. A published dataset of  5000 exonic SNPs associated with disease was analyzed to test the hypothesis that exon size-filtered sequence comparisons increase detection of splice-altering nucleotides. Improved splice predictions could be achieved when mutations occur at the third codon position, especially when a mutation decreases exon inclusion efficiency. The results demonstrate that coding pressures dominate nucleotide composition at invariable codon positions and that exon-size filtered sequence alignments permit identification of splice-altering nucleotides at wobble positions.</p>
  </div>
  
</div>
</li></ol>


</div>


	</div>

</section>


		</div>

	</div>


	<footer class="footer">

	<div class="footer__copyright">
		<span>© 2023 Fairlie Reese</span>
		<a href="https://jekyllthemes.io" target="_blank">Jekyll Themes</a>
	</div>

</footer>


	<!-- Javascript Assets -->
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
	<script src="/js/plugins-min.js"></script>
	<script src="/js/journal-min.js"></script>

	
	<!-- Extra Footer JS Code -->
	
	


</body>

</html>
